hendrix/test/performance/pyramid_with_crosstown_traffic.py

```diff
@@ -49,9 +49,10 @@ def cpu_heavy(heft, label=None):
             time.sleep(0)
 
         if count == end:
-            print("%s done: %s" % (label, time.time() - timer_start))
+            print "%s done: %s" % (label, time.time() - timer_start))
```

Comment:
This changed during a rebase that caused several problems. I just missed this one.